applications/serial_lte_modem/src/slm_settings.c

```diff
@@ -144,6 +155,7 @@ void slm_setting_fota_init(void)
 	fota_stage = FOTA_STAGE_INIT;
 	fota_status = FOTA_STATUS_OK;
 	fota_info = 0;
+	fota_reconnect = 1;
```

Comment:
Should this be really default on, even if the overlay has not been used?

Reply:
Yes, this helps to make LwM2M Registration happen automatically in case of NIDD.